lib/extensions.js

```diff
@@ -244,6 +268,14 @@ module.exports = function(realmConstructor) {
             }
         }
 
+        realmConstructor.Sync.prototype.uploadAllLocalChanges = function(timeout) {
```

Comment:
Shouldn't this be 
```suggestion
        realmConstructor.Sync.Session.prototype.uploadAllLocalChanges = function(timeout) {
```
instead?

Reply:
Ups, yes